ubuntudesign/developer.ubuntu.com
# Enable homepage as markdown file

## Done

Enable homepage as markdown file

## QA

- Run ./run to get started
- Go to http://0.0.0.0:8015/
- Make sure the copy is the same as the copy doc and that there are feeds from insights and tutorials

Doc: https://docs.google.com/document/d/1bvQDL9UlLMx8DFNQTCELPB4I3qBNwSV-OAig7z5-Jts/edit
